src/buffalonwb/metafile.yml

```diff
@@ -2,25 +2,25 @@ Behavior:
   EyeTracking:
     name: EyeTracking
     spatial_series:
-      data: true
-      description: eye tracking
-      name: SpatialSeries
-      reference_frame: ''
+      - data: true
```

Comment:
What is the purpose of the `data: true` statement

Reply:
data is one of the things we can't add through metadata, so the idea is to give an optional control to the user whether to include specific data or not. I think it could be useful with, say, multiple TimeSeries, and for a particular case the user decides to add just some of them

Reply:
But data is a required field of TimeSeries, so they shouldn't be able to set this to false because that would result in an invalid fine. I don't see how this Boolean flag helps users pick between datasets

Reply:
The way I pictured it was being used as a boolean control before creating or adding the objects to nwbfile:
```pyhton
if meta['Behavior']['Position']['spatial_series'][i]['data'] == True:
	make spatial_series object
	add it to nwbfile
```

looking at it now I agree it feels counter-intuitive that this control of add/don't add will be done by the `data` field. Anyway, it's just an idea of use for the `data` field on metadata, we can as well just remove it from GUI/metadata.

Reply:
I could make the Series group toggle active/inactive in the GUI with the checking/unchecking of the data checkbox, that could be a visually intuitive way to see what's going on.

Reply:
Yeah, I agree that sounds visually intuitive. I don't know if we should associate that with the `data` boolean flag though.

Reply:
I am removing the `data` field from metafiles and GUI. This is a general conversion point, not specific to this repo alone.